style.css

```diff
@@ -1,468 +1,1180 @@
+  }
+
+  .information__number-title {
+    font-size: 27px;
+  }
+
+  .information__number-subtitle {
+    font-size: 15px;
+    padding-top: 13px;
+    padding-left: 6px;
+  }
+
+  .information__number-text {
+    font-size: 10px;
+  }
+
+  .speed__foto {
+    width: 64px;
+    height: 73px;
+  }
+
+  .information__group-second {
+    justify-content: space-between;
+  }
+
+  .information__block__icon {
+    width: 160px;
+  }
+
+  .information__block__button {
+    width: 160px;
+    height: 40px;
+    margin-right: 0;
+  }
+
+  .button__text {
+    padding-left: 35px;
+  }
+
+  .section-first__arrow {
+    padding-top: 40px;
+  }
+
+  .section-second {
+    background-size: 100%;
+  }
+
+  .section-second__title {
+    padding-top: 51px;
+    font-size: 50px;
+    height: 357px;
+  }
+
+  .aside {
+    height: 100%;
+    margin: 0 auto;
+    padding-left: 45px;
+  }
+
+  .aside__it-about,
+  .information__group-second-aside {
+    display: none;
+  }
+
+  .section-third {
+    background-size: 100%;
+    background-position: 7px 73px;
+  }
+
+  .section-third__car {
+    height: 680px;
+  }
+
+  .aside-second {
+    height: 340px;
+  }
+
+  .aside-second__title {
+    width: 100%;
+  }
+
+  .aside-second__subtitle,
+  .aside-second__reting {
+    display: none;
+  }
+}
+
+@media (max-width: 380px) {
+  .header {
+    height: 48px;
+    padding: 27px 23px 0 4px;
+  }
+
+  .header__logo {
+    width: 138px;
+    height: 42px;
+  }
+
+  .header__burger img {
+    width: 24px;
+    height: 15px;
+    margin-top: 13px;
+  }
+
+  .section-first {
+    background-position: -17px 130px;
+    background-size: 421px;
+  }
+
+  .section-first__title {
+    padding-top: 49px;
+  }
+
+  .section-first__subtitle {
+    margin-top: -2px;
+  }
+
+  .content__img {
+    width: 100%;
+    margin-top: 48px;
+  }
+
+  .information {
+    max-width: 300px;
+  }
+
+  .information__block {
+    padding-top: 15px;
+  }
+
+  .time__foto {
+    width: 45.92px;
+    height: 50.16px;
+    padding-top: 20px;
+  }
+
+  .information__number-subtitle {
+    padding-top: 11px;
+    padding-left: 9px;
+  }
+
+  .information__number-text {
+    padding-left: 10px;
+  }
+
+  .speed__foto {
+    width: 48px;
+    height: 60px;
+    padding-top: 17px;
+    padding-left: 16px;
+    margin-bottom: 32px;
+  }
+
+  .information__block__icon {
+    width: 114px;
+    background: url("image/Test drive 1 (red 2).png") no-repeat;
+    background-position: center;
+    margin-top: 6px;
+  }
+
+  .information__block__button {
+    width: 160px;
+    height: 40px;
+    background: #e51937;
+  }
+
+  .button__text {
+    font-size: 13px;
+    text-transform: uppercase;
+    padding-top: 4px;
+    padding-left: 43px;
+  }
+
+  .section-first__arrow {
+    padding-top: 30px;
+  }
+
+  .section-second {
+    background: url("image/2013-tesla-model-s-cockpit (2).png") no-repeat;
+    background-position-y: 17px;
+    padding-top: 343px;
+  }
+
+  .title__text {
+    display: none;
+  }
+ 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)